roles/setup_server/tasks/main.yaml

```diff
@@ -43,8 +44,9 @@
         name:
           - python3-toml
   when:
-    - ansible_distribution in ['RedHat', 'CentOS']
-    - ansible_distribution_major_version|int == 9
+    - ansible_distribution in ['RedHat', 'CentOS', 'Fedora']
+    - ansible_distribution == 'Fedora' and ansible_distribution_major_version|int > 32 or
+      ansible_distribution_major_version|int == 9
```

Comment:
This is supposed to be specifically for RHEL9, I don't think Fedora should be included in this conditional block. It's possible Fedora needs it's own separate one or the conditional needs changing and the block name will as well

Reply:
I think renaming the block is the appropriate solution